lib/chef/resource/windows_certificate.rb

```diff
@@ -480,14 +474,12 @@ def export_cert(cert_obj, output_path:, store_name:, store_location:, pfx_passwo
         #
         def import_certificates(cert_objs, is_pfx, store_name: new_resource.store_name, store_location: native_cert_location)
           [cert_objs].flatten.each do |cert_obj|
-            # thumbprint = OpenSSL::Digest.new("SHA1", cert_obj.to_der).to_s
-            # pkcs = OpenSSL::PKCS12.new(cert_obj, new_resource.pfx_password)
-            # cert = OpenSSL::X509::Certificate.new(pkcs.certificate.to_pem)
             thumbprint = OpenSSL::Digest.new("SHA1", cert_obj.to_der).to_s
-            if is_pfx
-              if verify_cert(thumbprint) == true
-                Chef::Log.debug("Certificate is already present")
-              else
+            results = verify_cert(thumbprint)
```

Comment:
Seems like we could stick to `if verify_cert(thumbprint)....else...` instead of assigning to `results` and also checking `== false` (since verify_cert returns only true/false)

Reply:
I can see that. I was trying to simplify this. Rolled back a bit here.